teambit.bvm/bundle-env/env.jsonc

```diff
@@ -0,0 +1,172 @@
+{
+    /**
```

Comment:
I think you can just use the extends feature here instead of writing the entire env.jsonc